Code/GraphMol/Substruct/SubstructMatch.h

```diff
@@ -206,6 +209,21 @@ inline unsigned int SubstructMatch(ResonanceMolSupplier &resMolSupplier,
   return matchVect.size();
 };
 
+class RDKIT_SUBSTRUCTMATCH_EXPORT MolMatchFinalCheckFunctor {
```

Comment:
Since this is moving into the public API, it should pick up some documentation. Perhaps:
```
//! Class used as a final step to confirm whether or not a given atom->atom mapping is a valid
//! substructure match.
```

Reply:
Done